src/mono/netcore/System.Private.CoreLib/src/System/Environment.iOS.cs

```diff
@@ -74,10 +73,10 @@ private static string GetFolderPathCore(SpecialFolder folder, SpecialFolderOptio
                     return "/usr/share/templates";
 
                 case SpecialFolder.Fonts:
-                    return Path.Combine(Interop.Sys.SearchPath(NSSearchPathDirectory.NSDocumentDirectory), ".fonts");
+                    return CombineSearchPath(NSSearchPathDirectory.NSDocumentDirectory, ".fonts");
 
                 case SpecialFolder.Favorites:
-                    return Path.Combine(Interop.Sys.SearchPath(NSSearchPathDirectory.NSLibraryDirectory), "Favorites");
+                    return CombineSearchPath(NSSearchPathDirectory.NSLibraryDirectory, "Favorites");
```

Comment:
FYI, this is a functional change: if Interop.Sys.SearchPath did end up returning null, Path.Combine would throw.